### ISSUE TITLE
How We Work

### ISSUE THREAD

**Issue by sohilpandya** [fac-freelancers/website#108]
Designs and potential ideas related to how this section will work.

@heron2014 here are my initial thoughts of how it will work. 
![how-we-work](https://cloud.githubusercontent.com/assets/2305591/16399598/d525ac0e-3ccb-11e6-8046-210bc5d8365d.png)

Idea is that when they select each section there will be animation between it, so that the one selected will appear in front and text related to it will appear. 

What do you guys think? @minaorangina @shouston3

**Comment by heron2014** [comment]
@sohilpandya it looks good! I have few questions: these animated section would be on 100% width? Where the text will appear once you click on the section? 

Or is this computer part of the design ? haha

**Comment by sohilpandya** [comment]
Hmm. @heron2014 thats a good question. I was thinking the first one will be automatically be there by default and then they can select the 2/3 other sections to see the animations.

What were you thinking?

**Comment by iteles** [comment]
Might be a thought to figure out the content first and design something appropriate around that? :blush:

**Comment by sohilpandya** [comment]
@iteles yep, thats the next stage, trying to come up with the content for the site.

**Comment by iteles** [comment]
@sohilpandya But how do you know what the appropriate design for the site should be if you don't know what the content is?

**Comment by sohilpandya** [comment]
@iteles we've got a rough idea of what the how we work section will contain, it will be split up into 3/4 sections taking the client through the process that we have in place from start to finish. 
    - Concept / Deliverables
    - Development
    - Maintenance 

So we should be able to fit in the process with this kind of a flow, of course if it looks terrible then we can look for an alternative. But I think it'll look ok. 🏭

**Comment by iteles** [comment]
@sohilpandya Maintenance is an interesting one and content-wise I think each of these (particularly maintenance) warrant its own issue for debate ☺️ Sounds good!

**Comment by mcelearr** [comment]
@sohilpandya just in case you didn't notice, Sam asked me to put down some placeholder text for this section #78. All a bit of a guess because I don't actually know the working systems but hopefully you're able to use some of what I wrote.

**Comment by rub1e** [comment]
@/all I think the point Ines is trying to make, and the point that I've made in other issues, is that you don't start with a logo and decide what the company is about based on the logo - and you don't start with design and think about content separately. 

Imagine a client coming in and asking us to design a site before she'd put together any of the content...

**Comment by sohilpandya** [comment]
@rub1e I think I had a rough idea of what content I was going to be using before I came up with the sketches above. But we are coming up with the content as we speak.
